docs/creators/configure-forem.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Configuring Forem
+---
+
+#Basic Site Configuration Guide for Forem
+
+##Overview
+
+As a Forem admin, one of the first steps of managing your site will be to tailor your content, branding and other important details based on your community goals. This is made simple by the configuration page which can be found at https://<span></span>your.forem.url/internal/config.
+
+_We advise that you first complete the minimum set up before sending out your Forem link to your community._
+
+##Complete your Set Up, Configure your Forem
+
+Once your Forem instance is set up for the first time, you will most likely see the following banner:
+
+![Banner showing outstanding site configuration](https://dev-to-uploads.s3.amazonaws.com/i/2nosvfr7l47ymipmyh4o.png)
+
+This banner indicates that the minimum fields to configure your Forem have not been met.
+
+When you click on the link on banner, it will take you to the configuration page (i.e. https://<span></span>your.forem.url/internal/config).
+
+On this page you will see that the Get Started section is expanded. It contains all the mandatory fields that need to be filled out in order for the site to be in a usable state. Once it is filled out and submitted, the banner will then disappear.  
+
+##Access and Permissions
+
+The following permissions are required to be able to view and/or edit the config page:
+
+####Role: super\_admin
+When providing this role to a user they will be able to access the config page. However, this page will be a read only view for them. They will see the following:
+
+![Super Admin Permissions Role Provided](https://dev-to-uploads.s3.amazonaws.com/i/xpc8g9x46vzgi49ohc0d.png)
+
+####Role: single\_resource\_admin
+When providing this role to a user they will be able to access the config page, and they will be able to edit the config variables.
+
+![Super Resource Admin Role Provided](https://dev-to-uploads.s3.amazonaws.com/i/z5v2ou64imgqonmefolk.png)
+
+The first admin of your Forem will be given the highest permissions. Thereafter, they can provide the necessary permissions to subsequent users via the https://<span></span>your.forem.url/internal/permissions.
+
+##The Configuration Sections
+
+Currently, the configuration page is split into 3 sections. They are as follows:
+
+- A **Get Started section** that contains all required fields.
+  These are the minimum fields that required to be filled out, in order to get your Forem in a usable state.
```

Comment:
Whata about `These are the fields that are required to be filled out` ?

Reply:
✅